### ISSUE TITLE
RPMs for additional packages wrongly use settings for the primary package

### ISSUE THREAD

**Issue by ximon18** [NLnetLabs/ploutos#52]
From https://github.com/NLnetLabs/krill/issues/1003.

When `Cargo.toml` contains one or more sections named `[package.metadata.generate-rpm-alt-base-XXX]` Ploutos uses that as the new "base" settings by cutting it out and pasting it over `[package.metadata.generate-rpm]`.

However, the `xxx_install_script` settings in `Cargo.toml` are not used by Ploutos because it's not possible to refer to a file by name or path but instead the entire script must be included inline. Instead Ploutos instructs cargo-generate-rpm via its `--metadata-overwrite` command line argument to merge the contents of any `rpm_scriptlets_path` path that was provided as workflow input to Ploutos, irrespective of which package is being built.

**Comment by ximon18** [comment]
I've updated the Ploutos test suite to demonstrate the problem and also show that it is fixed in the coming v5.3.0 Ploutos release:

- `ploutos-testing` run [4003753737](https://github.com/NLnetLabs/ploutos-testing/actions/runs/4003753737/jobs/6872165729#step:15:51) shows the problem existing in the current v5.2.0 Ploutos release.
- `ploutos-testing` run [4004768904](https://github.com/NLnetLabs/ploutos-testing/actions/runs/4004768904) shows the problem being fixed in the pending v6.0.0 Ploutos release.

The test works by adding yet another job to the `ploutos-testing` workflow, each job being a single invocation of Ploutos with different settings to the other jobs. The new job, `test_with_alt_pkg`, configures Ploutos to build the usual `mytest` RPM package (there no DEB support yet in this job as it was focussed on fixing this issue) and adds a new `myalt` RPM package that should not have a systemd unit file or maintainer scripts. The new job has corresponding additional entries in `Cargo.toml` to name the package correctly and to refer to the correct assets. For RPMs systemd unit files have to be added as assets and the `myalt` RPM configuration deliberately doesn't refer to any.

Also, Ploutos has been updated to look for `rpm_scriptlets_path` differently if the package being built is an "alternate" package, i.e. its build matrix `pkg` name for the current package being built matches a config section in `Cargo.toml` of the form `[package.metadata.generate-rpm-alt-base-<PKG>]`. In this case it will look for `<rpm_scriptlets_path>-<PKG>`. Unfortunately this is a breaking change for anyone (however unlikely that is) who dependend on "alternate" RPM packages also using the same scriptlet files as the main package hence the need for a v6.0.0 release instead of a v5.3.0 release.
